src/ppx/application.cpp

```diff
@@ -1400,6 +1400,8 @@ void Application::DrawDebugInfo(std::function<void(void)> drawAdditionalFn)
     }
     uint32_t minWidth  = std::min(kImGuiMinWidth, GetWindowWidth() / 2);
     uint32_t minHeight = std::min(kImGuiMinHeight, GetWindowHeight() / 2);
+    // This is roughly centered for XR, upper-left corner for non-XR.
+    ImGui::SetNextWindowPos({GetWindowWidth() / 4.f, GetWindowHeight() / 4.f}, 0, {0.5f, 0.5f});
```

Comment:
I think the pivot value (0.5, 0.5) does not make sense for non-XR, and I also personally like the initial position better now (though that's subjective). You can try opening a non-XR application and resizing the debug window, it will resize from the center and the top and left side might go out of the window range. Maybe we should be setting these for XR only?

Reply:
Reworked this a good bit. Found the actual culprit so now it's doing something significantly more correct.